tailwind.config.js

```diff
@@ -10,7 +10,12 @@ module.exports = {
       green: defaultColors.green,
       red: defaultColors.red,
     },
+    maxWidth: {
+      ...tailwindConfigs.theme.maxWidth,
+      "prose-lg": "80ch",
+    },
```

Comment:
Merged. 

It would really help me if we could use `yarn link` for Fringe updates while developing though. I understand your concern about remembering to keep the release in sync, but adding things this way also adds things we have to keep track of, with the failure being pushed off to down the road. I know it's not ideal, but with so many apps, I think nothing will be.

Reply:
Setting up link now in the meantime this has been removed